pkg/newlog/Dockerfile

```diff
@@ -6,7 +6,7 @@ RUN eve-alpine-deploy.sh
 COPY ./  /newlog/.
 WORKDIR /newlog
 
-RUN GO111MODULE=on CGO_ENABLED=0 go build -ldflags "-s -w" -mod=vendor -o /out/usr/bin/newlogd ./cmd
+RUN GO111MODULE=on CGO_ENABLED=0 go build -ldflags "-s -w -X=main.Version=$(cat gopkgversion)" -mod=vendor -o /out/usr/bin/newlogd ./cmd
```

Comment:
```suggestion
RUN GO111MODULE=on CGO_ENABLED=0 go build -ldflags "-s -w -X=main.Version=$(<gopkgversion)" -mod=vendor -o /out/usr/bin/newlogd ./cmd
```

Reply:
It is a bashism. I originally had that, and spent easily 20-30 mins trying to figure out why the ldflag was blank. If I had submitted it right away, yetus would have told me, but I insisted on figuring it out myself first. 😆